outlines/react.md

```diff
@@ -0,0 +1,34 @@
+# React
+
+1. Introduction to React
+  1. Background: developed by Facebook, used widely in production, vibrant ecosystem
+  2. Installing and using React in Meteor 1.3
+  3. A simple example
+
+2. JSX in depth
+  1. What JSX really is (helping people understand it)
+  2. JSX workflow (transpiling - we've got it covered for you!)
+  3. JavaScript expressions inside JSX
+  4. Conditional and list rendering with JSX
+  5. Gotchas
+
+3. Components in depth
+  1. State
+  2. Props
+  3. Lifecycle
+  4. Passing callbacks
+  5. Stateless components
+
+4. Code Reuse
+  1. Mixins
+  2. Higher order components
+
+5. Getting Meteor Data into React Components
+  1. The `ReactMeteorData` mixin
+  2. The `ReactMeteorDataContainer` higher order component
+  3. Containers vs. presentational components
+  4. Optimizing re-renders with `shouldComponentUpdate`
+
+6. Routing with React
```

Comment:
I guess the thing that I really don't like is that we can't really say "you should follow the same principles, if not syntax, as we describe in the Routing guide article" since React Router is kind of threaded through all of your components, which goes against the idea of only accessing it at the top level? Although I guess you could say that all the components that are used by React Router should be containers?